airlock/views/request.py

```diff
@@ -723,3 +727,38 @@ def group_comment_delete(request, request_id, group):
                 messages.error(request, f"{field}: {error}")
 
     return redirect(release_request.get_url(group))
+
+
+@instrument(func_attributes={"release_request": "request_id", "group": "group"})
+@require_http_methods(["POST"])
+def group_comment_visibility_public(request, request_id, group):
+    release_request = get_release_request_or_raise(request.user, request_id)
+
+    form = GroupCommentDeleteForm(request.POST)
+
+    if form.is_valid():
+        comment_id = form.cleaned_data["comment_id"]
+        try:
+            bll.group_comment_visibility_public(
+                release_request,
+                group=group,
+                comment_id=comment_id,
+                user=request.user,
+            )
+        except exceptions.RequestPermissionDenied as exc:  # pragma: nocover
+            # currently, we can't hit this because of get_release_request_or_raise above.
+            # However, that may change, so handle it anyway.
+            raise PermissionDenied(str(exc))
+        except exceptions.FileNotFound:
+            raise Http404(
+                request, f"Comment not found in group {group} with id {comment_id}"
+            )
```

Comment:
Under what circumstances could this be raised?  As far as I can tell this would only happen if the comment had been deleted.  In which case I think the error message should be something like "Could not make comment public as it has been deleted".